src/client/handlers/agentLockAll.ts

```diff
@@ -18,4 +18,4 @@ class AgentLockAllHandler extends UnaryHandler<
   };
 }
 
-export { AgentLockAllHandler };
+export default AgentLockAllHandler;
```

Comment:
The names of the handlers was also updated. So `AgentLockAllHandler` should become `AgentLockAll`. This applies to all handlers

Reply:
ok i'll update the 63 handlers yet again

Reply:
You can do a find and replace to just remove the `Handler` from the names.

Reply:
> You can do a find and replace to just remove the `Handler` from the names.

That messes up all the imports, I need to refactor each one manually by renaming it